### DISCUSSION TITLE
Postgres DB compatibility

### DISCUSSION THREAD

**Discussion by adel-chouchane** [ApicurioArchive/apicurio-studio#2075] [Q&A] [answered]
What versions of postgres does Apicurio Studio supports ? Is it only postgres 9 ?

**Comment by adel-chouchane** [comment, marked as answer]
UPDATE : I tried using the latest version of postgres and it worked.

**Comment by EricWittmann** [comment]
Thanks for trying it @adel-chouchane !  We're not doing anything in the DB layer that should prevent newer versions of PG to work with Studio, but I'm happy to hear you've confirmed it.  We haven't had a chance to test using multiple versions of PG.
